src/app/routes/utils/regex/utils/index.js

```diff
@@ -58,7 +58,7 @@ export const getLiveRadioRegex = services => {
 
 export const getRadioAndTVRegex = services => {
   const serviceRegex = getServiceRegex(services);
-  return `/:service(${serviceRegex})/:serviceId(${mediaServiceIdRegex})/:mediaId(${mediaIdRegex}):amp(${ampRegex})?`;
+  return `/:service(${serviceRegex})/:serviceId(${mediaServiceIdRegex})(/programmes)?/:mediaId(${mediaIdRegex}):amp(${ampRegex})?`;
```

Comment:
Could we rename this to `getRadioRegex` - let's separate radio & TV because they are different and have slightly different URLs e.g. TV has `tv_programmes` instead of `programmes` in its URL

Reply:
```
const programmeRegex = 'tv_programmes|programmes'; 

return `/:service(${serviceRegex})/:serviceId(${mediaServiceIdRegex})(/programmeRegex)?/:mediaId(${mediaIdRegex}):amp(${ampRegex})?`;
```

We can do something like the above once the TV work is brought in

Reply:
Do we want to do that abstraction now (minus the `tv_programmes`) bit or just wait?

My only bugbear is that this regex is starting to get quite complicated, and I would rather have duplication + easy to understand over reusing a single regex. But just my opinion.

Reply:
Unfortunately, it wouldn't just be the regex we'd be duplicating - there's necessarily a one-to-one mapping between a route and the regex it matches against

If we introduce another regex, we'd have to duplicate the `onDemandRadio` directory here: https://github.com/bbc/simorgh/tree/latest/src/app/routes

Reply:
I'm not sure if this resolves any of this conversation but you can have multiple path regexes match for a route like shown here https://github.com/bbc/simorgh/blob/latest/src/app/routes/cpsAsset/index.js#L37

Reply:
I'm going to refactor the path regexes in a separate PR to keep the PRs readable. I have mocked up the TV routing so will be separating it into `getOnDemandRadioRegex` and `getOnDemandTVRegex`.